docs/Platforms.md

```diff
@@ -76,15 +76,44 @@ A marketing site by its nature will have fewer moving parts than a full-fledged
 
+### iOS Native Apps
 
+The high adoption rate of new Apple technology, whether it's the latest devices or the latest OS, demands that we keep up with the latest tools and practices. We develop iOS applications using *XCode* with the latest version of *Swift*.
+
+Instead of a simple MVC Architecture, we've found that the MVVM fits better with iOS views and view controllers. MVVM allows us to create applications that are easy to read, and scales well with an Agile method of software development.
+
+For package and version control, we use Cocoapods, which is the most common method for developers to publish their iOS code (as of this writing). Below are some pods that we use as the foundation of our mobile applications:
+
+* [RealmSwift](https://realm.io/docs/swift/latest/) - Realm offers a SQL-type store, similar to Core Data. It solves concurrency, like Core Data, with `Realm`s instead of `Context`s, but with a much easier interface and a simpler solution for creating and managing models. It also offers smooth decoding of JSON into a Realm objects, as long as the keys map directly.
+* [Moya](https://github.com/Moya/Moya) - Moya abstracts the network layer into a set of simple Swift config structs, where you define your routers, their HTTP methods, and even test/mock responses that you can unit test. It uses Alamofire as its core network layer.
+* [Locksmith](https://github.com/matthewpalmer/Locksmith) - Super simple library to abstract all your authentication keychain stuff.
+* [CocoaLumberjack/Swift](https://github.com/CocoaLumberjack/CocoaLumberjack) - This is a life savior for logging debug info into an emailable text file. Great for those production bugs that are hidden deep in background executions, or the ones that the crash reports provides *zero* help with.
 
 ### Cross Platform Apps
 
-### Native or Cross Platform?
+Facebook's [React-native](https://facebook.github.io/react-native/) currently offers the best cross-platform solution to develop iOS and Android applications, using the same-ish code base. Same-*ish* because you can easily create an entirely different experience for your Android application than your iOS application. Each platform comes with their own `index.js` file, and they have the *option* of sharing components and behaviors.
+
+The power of React-native is that you write interfaces that are translated to use the best native solutions under the hood. For example, iOS applications can use [NavigatorIOS](https://facebook.github.io/react-native/docs/navigatorios.html) which uses `UINavigationController` to give the same feel of a native iOS application.
+
+When you compile your react-native code base, a project for both iOS and Android is provided. The common code is compiled into libraries that each project consumes. You can inter-mix natively written iOS or Android components, along with components created through react-native.
+
+You'll also need either XCode or Android Studio to build and release the final product.
```

Comment:
Do we have recommended React Native libraries? I know we're using [NativeBase](https://nativebase.io) on one project.